Makefile

```diff
@@ -813,3 +813,9 @@ clean:
 	$(CMD_RM) -f .*.md5
 	$(CMD_RM) -f .check*
 	$(CMD_RM) -f .*-pkgs*
+
+# Run MkDocs development server to preview the documentation page
+.PHONY: mkdocs-serve
```

Comment:
we already have a makefile for docs: https://github.com/aquasecurity/tracee/blob/main/builder/Makefile.mkdocs

```
make mkdocs-build -f builder/Makefile.mkdocs
make mkdocs-serve -f builder/Makefile.mkdocs
```

Reply:
ok removed it :) I did not see this